lib/licenses.nix

```diff
@@ -109,6 +109,11 @@ in mkLicense lset) ({
     fullName = "Apache License 2.0";
   };
 
+  bitstreamVera = {
+    spdxId = "Bitstream-Vera";
+    fullName = "Bitstream Vera Font License";
+  };
```

Comment:
Not sure we can declare this a free license as it explicitly restricts modification if you don't rename it.

Normally this would be handled via trademarks, not the license (i.e. Rust), which would keep the license free but with a strict interpretation of the software freedoms, this could be considered unfree.

The OSD does explicitly allow this however: 

> The license may require derived works to carry a different name or version number from the original software.

https://opensource.org/osd

Reply:
Maybe a bit of an edge case but to me, this sounds like it the sources and the build product is still free and redistributable. Keep in mind that this licenses applies to fonts, which we usually do not patch and thus nixpgks/the binary cache does not modify the sources or the final product (unlike with binaries).

```
The Font Software may be modified, altered, or added to, and in
particular the designs of glyphs or characters in the Fonts may be
modified and additional glyphs or characters may be added to the
Fonts, only if the fonts are renamed to names not containing either
the words "Bitstream" or the word "Vera".
```